typing/printtyp.ml

```diff
@@ -2280,13 +2280,14 @@ let prepare_trace f tr =
 
 (** Keep elements that are [Diff _ ] and take the decision
     for the last element, require a prepared trace *)
```

Comment:
I wish the documentation string would be clear that the last element is *not* included in the first component of the returned value. (I would have guessed otherwise.)